.github/workflows/developer_onboarding_notification.yml

```
@@ -0,0 +1,73 @@
+
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
```

Prefer a more restrictive permission.
```suggestion
  permissions:
      pull-requests: write
```